bundlesize.json

```diff
@@ -2,7 +2,7 @@
   "files": [
     {
       "path": "./machine.min.js",
-      "maxSize": "1.38 kB"
+      "maxSize": "1.4 kB"
```

Comment:
Does this need to be bumped? There's no changes to machine.js now.

Reply:
Now Github Action is failing in the main branch because of this. I suspect that adding a check to the debug module by me earlier might have affected (but could not have been implemented otherwise)i